tutorials/single-deployment-tutorial.md

```diff
@@ -0,0 +1,96 @@
+# Terraform Google Cloud Migrate
```

Comment:
Let's focus the tutorial on the multi-deployment use case.

Reply:
Changed to multi deployment use case.